### PR TITLE
HIVE-27192 Use normal import instead of shaded import in TestSchemaTo…

### PR DESCRIPTION
**Pull request by zratkai**
…olCatalogOps.java

### What changes were proposed in this pull request?

Eliminated the shaded usage of com.google.common.io.Files;

### Why are the changes needed?
We do not need shaded usage here. 

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Jenkins test.

### CODE REVIEW

File: itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/tools/schematool/TestSchemaToolCatalogOps.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
 import org.apache.hadoop.hive.metastore.client.builder.PartitionBuilder;
 import org.apache.hadoop.hive.metastore.client.builder.TableBuilder;
 import org.apache.hadoop.hive.metastore.conf.MetastoreConf;
-import org.apache.hive.com.google.common.io.Files;
+import com.google.common.io.Files;
```
Comment (SourabhBadhya) [inline itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/tools/schematool/TestSchemaToolCatalogOps.java:38]: nit: After the removal of shading pattern, this does not follow alphabetical order of imports

**Review by SourabhBadhya — COMMENTED** [review]
LGTM +1, just 1 minor comment

**Review by SourabhBadhya — APPROVED** [review]

**Review by zabetak — COMMENTED** [review]
Are there more shaded references in the project?

Is there a way to prevent those from appearing again?

File: itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/tools/schematool/TestSchemaToolCatalogOps.java (hunk 1 above)
Comment (zratkai) [reply itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/tools/schematool/TestSchemaToolCatalogOps.java]: Done.

**Comment by zratkai** [comment]
@zabetak good question. I don't know how it can happen. I think autoimport do not do it. Can a code format check rule do that?

**Comment by zabetak** [comment]
@zratkai HIVE-22340 seems relevant can you please check?

**Comment by zratkai** [comment]
@zabetak thanks for this. I checked, and it is supposed to prevent shaded imports, but works where the import contains shaded:
 <bannedImport>**.shaded.**</bannedImport>
 
 If it does not contain, then it allows, as it happened in this case. I can add org.apache.hive.com.** and org.apache.hive.org.**,but is it enough?

**Comment by ayushtkn** [comment]
> If it does not contain, then it allows, as it happened in this case. I can add org.apache.hive.com.** and org.apache.hive.org.**,but is it enough?

still better than nothing, if we encounter some more instances, we can revisit and add them too :)

**Comment by zratkai** [comment]
I added add org.apache.hive.com.** and org.apache.hive.org.** to pom to avoid similar issue.

**Review by zabetak — APPROVED** [review]
LGTM!

**Review by ayushtkn — APPROVED** [review]
LGTM

**Pull request closed without merging**